docs/source/tutorials/configuring_noise.rst

```diff
@@ -0,0 +1,161 @@
+something up.
+
+pseudopeople has a noise type to simulate these sorts of responses for first and/or last names:
+:ref:`"Use a fake name." <use_a_fake_name>`
+With pseudopeople's default settings, this happens just 1% of the time.
+But let's say we're concerned it will happen more often in the future, and we want to see how
+robust our entity resolution methods are to this issue.
+
+Generating a simulated Current Population Survey
+------------------------------------------------
+
+Let's generate some 2025 Current Population Survey (CPS) data in a future scenario where 30% of people use fake last names on the survey.
+To do this, we will pass some **configuration** to the :func:`pseudopeople.generate_current_population_survey` function.
+
+.. code-block:: python
+
+    cps_2025 = psp.generate_current_population_survey(year=2025, config={
+        'current_population_survey': {
+            'column_noise': {
+                'last_name': {
+                    'use_fake_name': {
+                        'cell_probability': 0.3,
+                    },
+                },
+            },
+        },
+    })
+
+In English, our configuration says: in the **CPS** dataset, for the **last name** column, the **fake name** noise type's **cell probability**
+parameter should be 0.3 (30%).
+The full set of parameters for each noise type is documented at :ref:`Noise Type Details <noise_type_details>`.
+The :code:`column_noise` key specifies that we are configuring column-based noise; the categories of noise are explained in more detail
+on the :ref:`noise page <noise_main>`.
+
+Let's take a look at the names in our generated CPS dataset:
+
+.. code-block:: pycon
+
+    >>> cps_2025[['first_name', 'middle_initial', 'last_name']]
+       first_name middle_initial          last_name
+    0      Regina              S                COH
+    1         Pat              J              Posey
+    2   Stephanie              Y             Martin
+    3      Alaina              S               RESP
+    4       Haley              T                GOH
+    5     Katrina              V            Deltoro
+    6     Gregory              D             Talley
+    7   Catherine              R           Benjamin
+    8      Dustin              J           Benjamin
+    9        John              S              Bakal
+    10  Catherine              K            Trexler
+    11     Tamela              V             FEMALE
+    12       Reed              J                  F
+    13    William              D            Masters
+    14    Michael              D      Serna Marquez
+    15    Addison              I            Thomson
+    16     Denise              A     Harting-Conant
+    17      David              J                GOH
+    18       Mary              A  LADY OD THE HOUSE
+    19     Pamela              M               Reed
+    20       Alex              S             Ojukwu
+
```

Comment:
I know you mentioned it was distracting to have the fake names be all caps, and I thought you had a PR to change to title case in the code, right? Note however, that in title case, for example GOH becomes Goh, which is a real last name, so it's not necessarily obvious which ones are fake... It seems like people often just ignore case when doing linkage, by converting everything to lower case, for example...

Reply:
Yes -- the all-caps might be fixed by the release Steve just did. Let me check.

I agree that it seems like many practitioners ignore case in doing linkage. I don't think it was a mistake for us to output sensible capitalization, though perhaps it would be good if typos etc could mess it up. If people want to ignore case, they can.

I may reword a sentence here or there to reflect that it is no longer obvious which names are fake.